### DISCUSSION TITLE
Recommended Forge Build

### DISCUSSION THREAD

**Discussion by Tursiops-G** [Sinytra/ForgifiedFabricAPI#15] [Q&A] [answered]
I'm seeing recommendations to use Forge 4.7.1.3, and have experimented with 4.7.1.18 (Crashes in-game with 4.7.1.19 and up).

What is the Latest Recommended version of Forge for the current FFAPI Build?

Thanks.

**Comment by Su5eD** [comment, marked as answer]
On Minecraft 1.20.1, the recommended and stable Forge version for using FFAPI is **`47.1.3`**.

Starting with FFAPI version `0.86.1+1.8.1+1.20.1`, newer Forge versions will be rejected and users will be shown the FML loading error screen without the game crashing. This is the best way we can guarantee stability at the moment. Hopefully the situation improves in 1.20.2 and we'll be able to return to using recommended build versions.
